e2e/launchertester/basic_setup_test.go

```diff
@@ -45,6 +46,7 @@ func TestSetupWithCloudInit(t *testing.T) {
 	testCases := map[string]struct {
 		install_root     bool
 		withRegistryUser string
+		withWSL1         bool
```

Comment:
Am I missing something or is this never set to true in the below test cases?

Reply:
I think I lost something during rebases :)